### DISCUSSION TITLE
Feature Request - a way to see Kawaii Physics bodies in game after BeginPlay

### DISCUSSION THREAD

**Discussion by lordrex34** [pafuhana1213/KawaiiPhysics#124] [Ideas - 要望・アイデア]
It could be via a CVar console command. Would be really helpful.

**Comment by pafuhana1213** [comment]
Hi @lordrex34 ,
Does it help you? Please check it.
https://github.com/pafuhana1213/KawaiiPhysics/discussions/120

thanks!

**Reply by lordrex34** [reply]
yes, thanks this is exactly what I meant @pafuhana1213
